### PR TITLE
Exclude closed/dormant practices from count of practices

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: openprescribing/frontend/views/views.py [hunk 1]
```diff
@@ -206,8 +206,15 @@ def all_pcns(request):
 def pcn_home_page(request, pcn_code):
     pcn = get_object_or_404(PCN, ons_code=pcn_code)
     practices = Practice.objects.filter(pcn=pcn, setting=4).order_by("name")
+    num_open_practices = len([p for p in practices if p.status_code == "A"])
+    num_non_open_practices = len([p for p in practices if p.status_code != "A"])
```
Comment (evansd) [inline openprescribing/frontend/views/views.py]: You could avoid repeating the `status_code` with:
```python
num_non_open_practices = len(practices) - num_open_practices
```

File: openprescribing/templates/_practice_list.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<p>
+  There are {{ num_open_practices }} practices currently active in this CCG
```
Comment (evansd) [inline openprescribing/templates/_practice_list.html]: I think you need to replace `CCG` with `{% entity_type_human %}`, or whatever the variable is called, so this works with PCNs.

Comment (inglesp) [reply openprescribing/templates/_practice_list.html]: Ach, I had that in the first commit.  Well spotted.

**Comment by inglesp** [comment]
Fixes #1930.

**Pull request merged**